slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java

```diff
@@ -157,13 +164,26 @@ private void importBlocks(
             h,
             new tech.pegasys.web3signer.slashingprotection.dao.SignedBlock(
                 validator.getId(), jsonBlock.getSlot(), jsonBlock.getSigningRoot()));
+
+        minSlotTracker.trackValue(jsonBlock.getSlot());
       }
     }
+
+    if (minSlotTracker.compareTrackedValueTo(highestPreImportBlockSlot) > 0) {
```

Comment:
perhaps instead of compareTrackedValueTo and returning an int, would could just have something like if (minSlotTracker.isLessThan(highestPreImportBlockSlot)) { ... }